core/src/main/java/org/apache/accumulo/core/conf/Property.java

```diff
@@ -245,7 +245,8 @@
   TSERV_PREFIX("tserver.", null, PropertyType.PREFIX, "Properties in this category affect the behavior of the tablet servers"),
   TSERV_CLIENT_TIMEOUT("tserver.client.timeout", "3s", PropertyType.TIMEDURATION, "Time to wait for clients to continue scans before closing a session."),
   TSERV_DEFAULT_BLOCKSIZE("tserver.default.blocksize", "1M", PropertyType.BYTES, "Specifies a default blocksize for the tserver caches"),
-  TSERV_CACHE_POLICY("tserver.cache.policy", "LRU", PropertyType.STRING, "Specifies the eviction policy of the file data caches (LRU or TinyLFU)."),
```

Comment:
Yes.

And, before we get there, I am aware that this is technically not covered by our semver use. However, historically, we have treated configuration properties as such. We should not be dropping old, user-defined configuration keys without a deprecation cycle.